frontend/src/App.js

```diff
@@ -1,25 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import LandingPage from './pages/landing_page/LandingPage';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div className="App">
+        <LandingPage />
+      </div>
   );
 }
 
 export default App;
+
+
+// import logo from './logo.svg';
```

Comment:
not sure if this is intentional but because we have source control, we don't have to worry about preserving old code "just in case". we can always look back in the source tree (git) and see what the file used to look like and undo changes if we make mistakes!

tldr - remove these comments